Shopify/cli
# [Log Streaming] Removing references to event_type in log streaming.

### WHY are these changes introduced?

Fixes https://github.com/Shopify/shopify-functions/issues/171.
Part 2 of the CLI side of things. Part 1 found [here](https://github.com/Shopify/cli/pull/4035).


We renamed the `event_type` property in partners to `log_type`. The first CLI PR mentioned above was to support backwards and forward compatibility with [this](https://github.com/Shopify/partners/pull/55178) change in partners and avoid breaking changes for CLI users.

### WHAT is this pull request doing?


This PR will now cleanup any remaining references to the `event_type` property when polling for logs.

### How to test your changes?


Use this CLI branch to `shopify app dev` and make sure the logs are still streaming correctly.

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
